### PR TITLE
get from server some real data

### PR DESCRIPTION
**Pull request by shmuliko**
* add to model as can run raw sql
* fix sumByField as can change operator
* add boxing properties to dashboard display

### CODE REVIEW

File: protected/controllers/admin/index/index_index_controller.php [hunk 1]
```diff
@@ -12,13 +12,43 @@ public function content()
...
+    public function get_basic()
     {
-        $data = $this->model('server_load')->getByField('server_ip', SERVER, true, 'id DESC', 100);
+        $yesterdate = date('Y-m-d H:i:s', strtotime(gmdate('Y-m-d H:i:s') . ' - 1 day'));
+
+        $sql = ('
+            SELECT 
+                SUM(tq.delivered) delivered,
+                SUM(tq.impressions) impressions,
+                SUM(if(sp.status_id = 2, 1, 0)) + tq.converted converted
```
Comment (mosheka) [inline protected/controllers/admin/index/index_index_controller.php]: What is this + tq.converted?

File: protected/core/model.php [hunk 2]
```diff
@@ -410,10 +420,10 @@ public function countByField($field = null, $value = null)
      * @return number
      */
 
-    public function sumByField($sumField, $field = null, $value = null)
+    public function sumByField($sumField, $field = null, $value = null, $operator = null)
```
Comment (mosheka) [inline protected/core/model.php]: Why not using default value 
$operator  = "=" 
instread of having if below?

File: protected/core/model.php [hunk 3]
```diff
@@ -410,10 +420,10 @@ public function countByField($field = null, $value = null)
...
     {
         $stm = $this->pdo->prepare('
-        SELECT SUM(' . $sumField . ') sm FROM ' . $this->table . ( $field ? ' WHERE ' . $field . ' = :' .$field : '' ) . '
+        SELECT SUM(' . $sumField . ') sm FROM ' . $this->table . ( $field ? ' WHERE ' . $field . ($operator ? ' ' . $operator : ' =') . ' :' . $field : '' ) . '
```
Comment (mosheka) [inline protected/core/model.php]: Here you can avoid the if

File: protected/templates/admin/en/index/index.php [hunk 4]
```diff
@@ -42,7 +42,17 @@
                     </div>
                     <div class="dashboard-item dashboard-color-9" id="dashboard-uuss">
                         <p>UUSS<p>
-                        <p></p>
+                        <table>
+                            <tr><td>Quentity</td><td id='uuss-qnt'></td></tr>
```
Comment (mosheka) [inline protected/templates/admin/en/index/index.php]: Quantity
Please google before you enter text

File: protected/templates/admin/en/index/index.php [hunk 5]
```diff
@@ -64,10 +74,35 @@
...
+                var viewability = 0;
+                if(Number(data.impressions) > 0 && Number(data.delivered) > 0){
+                    viewability = (100 * data.impressions / data.delivered).toFixed(2);
+                }
+                $('#dashboard-total-viewability p:last').html(viewability + '&percnt;');
+                $('#dashboard-clicks p:last').html(data.clicks || '-');
+                var ctr = 0;
+                if(Number(data.clicks) > 0 && Number(data.delivered) > 0){
+                    ctr = (100 * data.clicks / data.delivered).toFixed(2);
+                }
```
Comment (mosheka) [inline protected/templates/admin/en/index/index.php]: better be done in a single line:
ctr = (... ) ? ... : 0;

File: protected/controllers/admin/index/index_index_controller.php (hunk 1 above)
Comment (shmuliko) [reply protected/controllers/admin/index/index_index_controller.php]: i copy from yevgeniy at field data
is cached clicks plus col from task_queue.converted

File: protected/core/model.php (hunk 2 above)
Comment (shmuliko) [reply protected/core/model.php]: i change this

File: protected/core/model.php (hunk 3 above)
Comment (shmuliko) [reply protected/core/model.php]: changed

File: protected/templates/admin/en/index/index.php (hunk 4 above)
Comment (shmuliko) [reply protected/templates/admin/en/index/index.php]: changed
most times i copy from other places in application or from trello

File: protected/templates/admin/en/index/index.php (hunk 5 above)
Comment (shmuliko) [reply protected/templates/admin/en/index/index.php]: in first wish is typed in one line, but is not readable

**Pull request merged**